mantidproject/mantid
# Indirect Data Reduction Calibration -  Peak Min/Max range bars

The range bars in `Calibration` section of the `Indirect Data Reduction > ISIS Calibration` interface should now update properly on data load and they now work for OSIRIS graphite 002.

**To test:**
- Change default facility and instrument to ISIS / IRIS (View > Preferences > Mantid)
- Open `ISIS Calibration` (Interfaces > Indirect > Data Reduction > ISIS Calibration)
- Ensure the Instrument is set to `IRIS / graphite / 002`
- Run no = `26176`
- Ensure the curve loads into the Calibration mini plot and the blue range bars (peak) contain the peak and the green range bars (background) should be slightly higher the peak.
- The plot should look like: 
  ![image](https://cloud.githubusercontent.com/assets/13132128/15672046/1454283a-2726-11e6-8067-7c50acf5b133.png)
- Change the Instrument to `OSIRIS / graphite / 002`
- Run No = `121313`
- Ensure that the above statements about the range bars are true for OSIRIS data
- The plot should look like:
  ![image](https://cloud.githubusercontent.com/assets/13132128/15672125/813b4b2c-2726-11e6-9807-b1dc3f913bcb.png)

Fixes #16392

[RELEASE NOTES](https://github.com/mantidproject/mantid/blob/40ff87971c0eaf8a63d53348f7227cf3ece8ea9b/docs/source/release/v3.7.0/indirect_inelastic.rst#bugfixes)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
